client/src/main/scala/drt/client/components/TerminalPlanningComponent.scala

```diff
@@ -167,7 +171,8 @@ object TerminalPlanningComponent {
                 MuiRadioGroup(row = true)(^.value := state.timePeriod, ^.onChange ==> ((e: ReactEventFromInput) => {
                   scope.modState(_.copy(timePeriod = e.target.value.toInt)) >>
                     Callback(SPACircuit.dispatch(UpdateUserPreferences(userPreferences.copy(userSelectedPlanningTimePeriod = e.target.value.toInt)))) >>
-                      Callback(SPACircuit.dispatch(GetForecastWeek(props.page.dateFromUrlOrNow, Terminal(props.page.terminalName), e.target.value.toInt)))
+                      Callback(SPACircuit.dispatch(GetForecastWeek(props.page.dateFromUrlOrNow, Terminal(props.page.terminalName), e.target.value.toInt))) >>
+                      Callback(GoogleEventTracker.sendEvent(props.page.terminalName,"planning_timePeriod", e.target.value))
```

Comment:
Can we be consistent with formatting - we have `planning_timePeriod` ie a mix of underline and case, but others have used dashes so it should probably be `planning-time-period`